sdk/helios_processor.go

```diff
@@ -0,0 +1,24 @@
+package sdk
+
+import (
+	"context"
+
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/baggage"
+	"go.opentelemetry.io/otel/sdk/trace"
+)
+
+const HELIOS_TEST_TRIGGERED_TRACE = "hs-triggered-test"
+
+type HeliosProcessor struct{}
```

Comment:
Why did you define this struct?
Can't you call the functions here without an input?